package.json

```diff
@@ -61,7 +61,7 @@
     "del": "^5.1.0",
     "elf-cam": "^0.1.1",
     "end-of-stream": "^1.4.4",
-    "esbuild": "^0.11.0",
+    "esbuild": "0.11.10",
```

Comment:
:'(

Reply:
FYI, I inadvertently updated `esbuild` when I merged https://github.com/netlify/zip-it-and-ship-it/pull/460/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519L4305.

Leaving it as is, as this PR should fix that